www/blog/2023/06/19/karmashapes/index.md

```diff
@@ -0,0 +1,276 @@
+---
+layout: page
+title: Introducing Karmashapes
+published: false
+date: 2023-06-19
+permalink: /blog/2023/06/19/introducting-karmashapes/
+category: blog
+excerpt: "Thanks to the Karmashapes initiative, Who’s On First now provides the best open data for towns and villages in India."
+authors: [justinelliotmeyers, stepps00, nvkelso]
+image: "images/coverage_overview.png"
+tag: [whosonfirst,wof,data,import,india]
+---
+
+[![](images/249689_3cfb5bacb30f607a_b.jpg)](https://collection.cooperhewitt.org/objects/18412045/)
+
+<div style="font-size:small;font-style:italic;text-align:center;">
+Printing Block (India); wood; 1957-123-6 <a href="https://collection.cooperhewitt.org/objects/18412045/">Collection of Cooper Hewitt Museum</a>.</div>
+
+
+# Introducing Karmashapes
```

Comment:
Remove this headline entirely, as the WOF blog machine will add it automatically for you 

(else there's 2 headlines in the post, one auto generated at the top and your manual one here)